datacenterlight/templates/datacenterlight/includes/_calculator_form.html

```diff
@@ -1,4 +1,14 @@
 {% load staticfiles i18n%}
+{% if vm_pricing %}
+<script type="application/javascript">
+    window.vat_inclusive = {% if vm_pricing.vat_inclusive %}true{% else %}false{% endif%};
+    window.vat_percentage = {% if vm_pricing.vat_percentage %}{{vm_pricing.vat_percentage}}{% else %}0{% endif%};
```

Comment:
can use the `default` filter for a shorter syntax as `{{vm_pricing.vat_percentage|default:0}}`